Cargo.toml

```
@@ -389,6 +389,16 @@ description = "Demonstrates how to prevent meshes from casting/receiving shadows
 category = "3D Rendering"
 wasm = true
 
+[[example]]
+name = "skybox"
+path = "examples/3d/skybox.rs"
+
+[package.metadata.example.skybox]
+name = "Skybox"
+description = "Load a cubemap texture onto a cube like a skybox and cycle through different compressed texture formats. Note that the `ktx2` and `zstd` features are needed to be able to load the KTX2 textures."
```

```suggestion
description = "Load a cubemap texture onto a cube like a skybox and cycle through different compressed texture formats."
```
if `required-features` is set on the example, no need to add the note